EIPS/eip-4907.md

```diff
@@ -0,0 +1,185 @@
+---
+eip: 4907
+title: ERC-721 User And Expires Extension
+description: Standard interface extension for ERC-721 user and expires.
```

Comment:
I don't think this description provides any additional information over the title. I might suggest:

```suggestion
description: Add a time-limited role with restricted permissions to ERC-721 tokens
```